### PR TITLE
Tanium server_access, full import and data handler

### PR DESCRIPTION
**Pull request by JingqiuDu**

### CODE REVIEW

File: connectors/tanium/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## Connector
+
+Install python dependencies
+```
+pip3 install -r requirements.txt
+```
+
+Running the connector:
+```
+python3 app.py -server=http://localhost:8000 -username=admin -password=admin -car-service-url=<car-service-url> -car-service-key=<car-service-key> -car-service-password=<car-service-password> -source=<source>
```
Comment (omkar-g) [inline connectors/tanium/README.md]: Update the docs

File: connectors/tanium/configurations/config.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            "displayName": "Tanium",
+            "group": "tanium"
+        },
+        "host": {
+            "type": "text",
+            "regex": "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\\-]*[A-Za-z0-9])$",
+            "optional": true,
+            "default": ""
+        }
+    },
```
Comment (omkar-g) [inline connectors/tanium/configurations/config.json:18]: Add port field here

File: connectors/tanium/configurations/lang.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (omkar-g) [inline connectors/tanium/configurations/lang.json]: Delete this file

File: connectors/tanium/configurations/lang_en.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+    "connection": {
+        "host": {
+            "label": "Management IP address or Hostname",
+            "placeholder": "app.randori.io",
+            "description": "Specify the IP address or  hostname of the data source so that IBM Cloud Pak for Security can communicate with it"
+        }
+    },
```
Comment (omkar-g) [inline connectors/tanium/configurations/lang_en.json:7]: Add port here too

File: connectors/tanium/connector/data_handler.py [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+        res = {}
+        for field in fields:
+            res[field] = obj[field]
+        return res
+
+    # Create asset Object as per CAR data model from data source
+    def handle_assets(self, obj):
+        res = {}
+        res['external_id'] = obj['id']
+        res['name'] = "%s, %s" % (obj['manufacturer'], obj['name'])
```
Comment (omkar-g) [inline connectors/tanium/connector/data_handler.py]: Throwing error on "manufacturer"

File: connectors/tanium/connector/server_access.py [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+                            name
+                          },
+                          lastLoggedInUser,
+                          isVirtual,
+                          isEncrypted,
+                          chassisType,
+                          os {
+                            name
+                          },
+                          disks {
```
Comment (omkar-g) [inline connectors/tanium/connector/server_access.py]: I feel we dont need this

File: connectors/tanium/connector/server_access.py [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+                            free
+                            total
+                            usedPercentage
+                            usedSpace
+                          },
+                          eventCounts{
+                            network
+                          },
+                        discover {
+                          natIpAddress,
```
Comment (omkar-g) [inline connectors/tanium/connector/server_access.py]: What is natIpAddress

File: connectors/tanium/requirements.txt [hunk 8]
```diff
@@ -0,0 +1 @@
+car-connector-framework==3.0.0
```
Comment (omkar-g) [inline connectors/tanium/requirements.txt:1]: 3.0.1

File: connectors/tanium/connector/server_access.py (hunk 7 above)
Comment (JingqiuDu) [reply connectors/tanium/connector/server_access.py]: I deleted it, seems like we don't need it

**Pull request merged**